core/controllers/acl_decorators.py

```diff
@@ -2753,18 +2753,17 @@ def test_can_accept_suggestion(
     return generate_decorator_for_handler
 
 
-def can_update_suggestions(handler):
-    """Decorator to check whether user can update the suggestions that
-    they are allowed to review.
+def can_reviewer_or_suggester_update_suggestions(handler):
+    """Decorator to check whether user can update the suggestions.
```

Comment:
Suggest: ``...whether the current user can update...``

Reply:
Done